meta-sokol-flex-staging/recipes-kernel/lttng/lttng-tools/0001-Ensure-that-the-consumerd-configure-arguments-are-us.patch

```diff
@@ -8,6 +8,8 @@ arguments are not being used by lttng-sessiond, which causes problems
 with providing a well configured multilib tracing environment. Ensure
 that they are used as the default values in sessiond.
 
+Upstream-Status: Inappropriate [OE specific]
```

Comment:
I don't believe this is the correct status. This should probably be upstreamed, or at least discussed with upstream. Both lttng proper and oe-core.

Reply:
@kergoth since you're the author of this patch, can you initiate the discussion. I'll update the status.

Reply:
I am merging it. we will handle it in cleanup task